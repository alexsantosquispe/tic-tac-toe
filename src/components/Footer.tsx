export const Footer = () => {
  return (
    <footer className="flex h-14 w-full items-center justify-center border-t border-neutral-200 bg-white px-8 text-xs text-neutral-600">
      Â© 2025 TicTacToe by AlexSan. All rights reserved.
    </footer>
  );
};
