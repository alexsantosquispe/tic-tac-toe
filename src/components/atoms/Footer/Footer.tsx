export const Footer = () => {
  return (
    <footer className="flex h-14 w-full items-center justify-center border-t border-neutral-200 px-8 text-[0.8125rem] font-light dark:border-white/20">
      <p className="text-center">
        2025 TIC-TAC-TOE by{' '}
        <a
          href="https://github.com/alexsantosquispe"
          target="_blank"
          rel="noopener noreferrer"
          className="font-medium text-blue-600 transition-colors duration-200 hover:text-blue-800 dark:text-blue-400 dark:hover:text-blue-300"
        >
          Alex Santos
        </a>
        . All rights reserved.
      </p>
    </footer>
  );
};
