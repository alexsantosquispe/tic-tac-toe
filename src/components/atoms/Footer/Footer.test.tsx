import { render } from '@testing-library/react';
import { Footer } from './Footer';

describe('Footer', () => {
  it('should render the component correctly', () => {
    const component = render(<Footer />);

    expect(component).toMatchSnapshot();
  });
});
