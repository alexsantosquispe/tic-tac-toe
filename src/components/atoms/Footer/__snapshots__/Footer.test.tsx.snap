// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Footer should render the component correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <footer
        class="flex h-14 w-full items-center justify-center border-t border-neutral-200 px-8 text-[0.8125rem] font-light dark:border-white/20"
      >
        <p
          class="text-center"
        >
          2025 TIC-TAC-TOE by
           
          <a
            class="font-medium text-blue-600 transition-colors duration-200 hover:text-blue-800 dark:text-blue-400 dark:hover:text-blue-300"
            href="https://github.com/alexsantosquispe"
            rel="noopener noreferrer"
            target="_blank"
          >
            Alex Santos
          </a>
          . All rights reserved.
        </p>
      </footer>
    </div>
  </body>,
  "container": <div>
    <footer
      class="flex h-14 w-full items-center justify-center border-t border-neutral-200 px-8 text-[0.8125rem] font-light dark:border-white/20"
    >
      <p
        class="text-center"
      >
        2025 TIC-TAC-TOE by
         
        <a
          class="font-medium text-blue-600 transition-colors duration-200 hover:text-blue-800 dark:text-blue-400 dark:hover:text-blue-300"
          href="https://github.com/alexsantosquispe"
          rel="noopener noreferrer"
          target="_blank"
        >
          Alex Santos
        </a>
        . All rights reserved.
      </p>
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
